umbraco/Umbraco-CMS
# V14: Add optional key to template create model

# Notes
- Adds key to `CreateTemplateRequestModel`, so you can specify keys when creating.
- Added optional template key to ITemplateService.

# How to test
- Using swagger
- Create a template with a key
- Try the `Get` endpoint with the key you created the item with, this should give you an item 👏
